lib/upgrade/upgrade_supported.go

```diff
@@ -47,22 +47,24 @@ var insecureHTTP = &http.Client{
 	},
 }
 
-// LatestGithubReleases returns the latest releases, including prereleases or
+// FetchLatestReleases returns the latest releases, including prereleases or
 // not depending on the argument
-func LatestGithubReleases(releasesURL, version string) ([]Release, error) {
```

Comment:
gofmt says remove the parenthesis around the return type when there's only one, otherwise this LGTM